ios/Nightly-Info.plist

```diff
@@ -51,6 +51,8 @@
 	<string>Enabling camera allows you to scan QR codes.</string>
 	<key>NSFaceIDUsageDescription</key>
 	<string>Enabling Face ID allows you quick and secure access to your account.</string>
+	<key>NSLocationAlwaysUsageDescription</key>
+	<string>One of the software dependencies of Yoroi mobile requires this permission, but Emurgo does not use neither store this information</string>
```

Comment:
We've been having a warning related to this each time we push to the App Store, since I work on this project. Not sure what is the actual dependency that needs this, so I added a generic message.